### ISSUE TITLE
Detach process

### ISSUE THREAD

**Issue by mickaelmonsieur** [Wyliodrin/pybass#1]
Hi,
I already worked with bass.dll, but under Delphi 7 on Win2K, +10 years ago.
I have a problem today with Python, I start on this language.
I created a wx GUI, and when I press a button, an mp3 file is played. Nice!
The problem is that as long as the file is not finished, the button remains pressed and the program freezes (I'm on Mac OS and the OS tells me "This program does not answer")
I must definitely detach the process, but how? with a Thread?
Thanks,
Mickael

**Comment by alexandruradovici** [comment]
Yes, a thread would be good idea. You can download [WyliodrinSTUDIO](https://github.com/Wyliodrin/WyliodrinSTUDIO), create a new Visual project and drag the blocks for multimedia. That should show you the code.
